.travis.yml

```diff
@@ -94,16 +94,23 @@ matrix:
         # time. Since we test the latest Numpy as part of the builds with all
         # optional dependencies below, we can focus on older builds here.
         # Numpy 1.11 is tested below in the image tests, 1.12 in the Initial stage test.
+        # Run this test using native pytest
         - os: linux
           env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.10
-               SETUP_CMD='test --open-files'
+               INSTALL_CMD='python setup.py build_ext --inplace'
+               TEST_CMD='pytest --open-files --doctest-rst'
+          script:
+            - echo "$INSTALL_CMD"
```

Comment:
Why are these echos needed here? Travis lists these env variable at the top of the log file anyway, e.g. https://travis-ci.org/astropy/astropy/jobs/298131620#L498

Reply:
I'll remove these. There's also one at the bottom under `script:` which was not introduced by this PR.

Reply:
yes, please remove that one, too. If any more verbosity is needed, one can always use the ``DEBUG=True`` setting